pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
Do you think I should spend the time to update and test the routes for block, filesystem and object deletion to use `DELETE` verb and URI query params, instead of getting onto v0.4 implementations for #399, #397, #433, and #454?